build.sbt

```diff
@@ -145,7 +147,8 @@ lazy val `java-common` = project
   .settings(
     name := "otel4s-java-common",
     libraryDependencies ++= Seq(
-      "io.opentelemetry" % "opentelemetry-api" % OpenTelemetryVersion
+      "org.typelevel" %%% "cats-effect-kernel" % CatsEffectVersion,
+      "io.opentelemetry" % "opentelemetry-sdk" % OpenTelemetryVersion
```

Comment:
These are stronger dependencies, but I don't have a clear reason why we _couldn't_ use the SDK and Cats Effect here.